StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsSynchronize.swift

```diff
@@ -22,54 +21,3 @@ struct FinancialConnectionsSynchronize: Decodable {
         let merchantLogo: [String]
     }
 }
-
-struct FinancialConnectionsConsent: Decodable {
```

Comment:
all of these were moved into separate files/models because when searching for a specific model, Xcode wouldn't be great at navigating _right_ where the definition is because multiple definitions were stacked in the same file

(sry for the extra unrelated code, but it should be still easy to analyze/double-check as the logic of this PR is simple)